### PR TITLE
Line sensor updated to print 'b' and 'w'

### PR DESCRIPTION
**Pull request by karan259**

### CODE REVIEW

**Comment by karan259** [comment]
@CleoQc can you check this

File: Software/Python/easygopigo.py [hunk 1]
```diff
@@ -470,7 +470,23 @@ def read_raw_sensors(self):
         _wait_for_read()
 
         _grab_read()
+        
         five_vals = line_sensor.read_sensor()
+        five_vals=[0]*5
+        white_line=line_sensor.get_white_line()
+        black_line=line_sensor.get_black_line()
```
Comment (CleoQc) [inline Software/Python/easygopigo.py]: black_line seems to be unused.

File: Software/Python/easygopigo.py [hunk 2]
```diff
@@ -470,7 +470,23 @@ def read_raw_sensors(self):
...
+        range_sensor= line_sensor.get_range()
+        threshold=[a+b/2 for a,b in zip(white_line,range_sensor)]
+        # print (white_line,black_line,range_sensor,threshold)
+        raw_vals=line_sensor.read_sensor()
```
Comment (CleoQc) [inline Software/Python/easygopigo.py]: This line is the only one that should be between the _wait_for_read/_grab_read and the _release_read()

File: Software/Python/easygopigo.py (hunk 1 above)
Comment (karan259) [inline Software/Python/easygopigo.py]: will remove the other one

**Comment by CleoQc** [comment]
Not merging this but thank you @karan259  as the code was very helpful.

**Pull request closed without merging**